### ISSUE TITLE
Constant nag popup asking "Ready to support your favorite sites?"

### ISSUE THREAD

**Issue by basildane** [brave/browser-laptop#12758]
### Description
Brave browser displays unwanted nag screen, repeatedly.
Text: "Hello! Ready to support your favorite sites?  Brave will fill your wallet with tokens to get you started!"

The only options are "maybe later" or "I'm ready".  The "NO!" option is missing.

### Steps to Reproduce
  1.  Open Brave browser.
  2.  Be annoyed.

**Actual result:**

**Expected result:**

**Reproduces how often:**
Seems to occur every day on every installation of Brave.

### Brave Version

**about:brave info:**
 
Brave: 0.19.139 
rev: 99aa17da28e040543b4ba217d2c4dd6633bf1446 
Muon: 4.7.2 
libchromiumcontent: 64.0.3282.85 
V8: 6.4.388.20 
Node.js: 7.9.0 
Update Channel: Release 
OS Platform: Microsoft Windows 
OS Release: 10.0.16299 
OS Architecture: x64

**Reproducible on current live release:**
Yes.

**Comment by BraveSupporter** [comment]
I don't seem to have this issue.

**Comment by basildane** [comment]
Interesting, I have Brave installed on 6 computers at home and work, and it does this on all six of them.

**Comment by bsclifton** [comment]
cc: @bradleyrichter

**Comment by bradleyrichter** [comment]
@NejcZdovc Can you post the current logic here so we can examine and adjust it accordingly?

**Comment by NejcZdovc** [comment]
as far as I can tell from the notification this is triggered when we add promotion. We added disable option in 0.20 https://github.com/brave/browser-laptop/issues/12313

**Comment by bradleyrichter** [comment]
#12313 is only for BAT promotions. General Brave Payments invitations are separate. 

Those originally had a time restriction before first appearance. And a longer delay before appearance 2. 

I would like to use "no thanks" for the deny button on the second attempt which would disable further notifications until the user would opt-in to BP.

**Comment by NejcZdovc** [comment]
yeah but if you check out initial issue ` Brave will fill your wallet with tokens to get you started!` it's about BAT promotion

**Comment by bradleyrichter** [comment]
Ah...yes. You are right. This mixed alert will be handled by the new switch for BAT offers.

**Comment by srirambv** [comment]
+1 from https://community.brave.com/t/stop-asking-about-payments/14797/2

**Comment by dfwdraco76** [comment]
My only options are "Maybe Later" and "I'm Ready".  I've clicked "Maybe Later" at least a dozen times since installing beta 0.20.22 a day or two ago.

![capture](https://user-images.githubusercontent.com/30734939/35484214-844885da-0412-11e8-98e7-eb0dfd605e63.PNG)

**Comment by basildane** [comment]
Can I make a suggestion to the devs?  Rather than code ways to stop this message, you need to rethink some things.  As in: don't do this in the first place.  These types of messages are nothing but a nuisance and this type of behavior is what has driven the community away from other applications and websites.  If someone wants to do something, they will.  If they don't, then no amount of nagging is going to change that.  It's simply going to make them leave, which I'm sure is not your intention.

**Comment by bradleyrichter** [comment]
I would normally agree ^ but the current alert approach is working for UGP grant acceptance. We will improve Brave Payments awareness through better marketing and a better onboarding for new users. At that point we can reevaluate removal of nag alerts.

To solve this nag for the majority, this is what we need to do:

Show the first time at 10 days of browsing.
If answer is “maybe later”, then show again at 30 days. 

On this 2nd nag, the button changes from “maybe later” to “no thanks” and never nags again.


@NejcZdovc please reference this issue in the PR for this fix...

**Comment by bsclifton** [comment]
Moving back to 0.22.x for now (since there isn't already a PR for this)

**Comment by LaurenWags** [comment]
@NejcZdovc @bradleyrichter confused on this one - per https://github.com/brave/browser-laptop/issues/12758#issuecomment-364570120 I get the sense (but it's not explicitly stated) that the 'No Thanks' button should not be displayed on the first appearance of this notification. However, per the test plan in https://github.com/brave/browser-laptop/pull/13465 I am seeing this 'No Thanks' button on the first appearance of the notification. Can one of you please clarify - should 'No Thanks' be displayed the first time or not?

![screen shot 2018-03-22 at 10 55 13 am](https://user-images.githubusercontent.com/28145373/37778113-83c02592-2dbf-11e8-81b2-8f596be3008e.png)

**Comment by LaurenWags** [comment]
second question @nejczdovc and @bradleyrichter - should the 'No Thanks' also be on this notification:

![screen shot 2018-03-22 at 11 01 16 am](https://user-images.githubusercontent.com/28145373/37778505-7aafaf62-2dc0-11e8-92c4-4ccf445b79d3.png)

**Comment by NejcZdovc** [comment]
@LaurenWags it should be on all promotion notifications

**Comment by NejcZdovc** [comment]
@LaurenWags yes we should display it for the first time as well

**Comment by LaurenWags** [comment]
so then 'Maybe Later' does not change to 'No Thanks', correct? @nejczdovc

**Comment by NejcZdovc** [comment]
maybe later will show notification again in 24h, where no thanks will disable promotions @LaurenWags

**Comment by bradleyrichter** [comment]
yes @LaurenWags - we didn't correctly update the test case.

We decided to reduce NAG pain by giving the user more control on the first notice. 

Eventually the suggested choice on the far right will be an orange button as well.

**Comment by LaurenWags** [comment]
Thanks for the confirmations @nejczdovc and @bradleyrichter!

**Comment by LaurenWags** [comment]
Verified on macOS 10.12.6 x64 using the following build:
* 0.22.7 https://github.com/brave/browser-laptop/commit/8bb7e77f69bca186429aafda4bc6ba4d82a5de47
* libchromiumcontent: 65.0.3325.181
* muon: 5.1.1
